notebooks/network_mapping/Supervised_Asset_Classification.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>This is a bit messy, as is all the outputs above it. I suggest you consolidate this all into one block and write brief explanatory text for the cells. What are we seeing and what do we expect to see?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>